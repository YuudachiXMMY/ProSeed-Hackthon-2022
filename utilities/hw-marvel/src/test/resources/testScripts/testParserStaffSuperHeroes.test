# Test Marvel parser to load StaffSuperHeroes.tsv correctly

LoadGraph staffHeroes staffSuperheroes.tsv

ListNodes staffHeroes
ListChildren staffHeroes Ernst-the-Bicycling-Wizard
ListChildren staffHeroes Grossman-the-Youngest-of-them-all
ListChildren staffHeroes Notkin-of-the-Superhuman-Beard
ListChildren staffHeroes Perkins-the-Magical-Singing-Instructor