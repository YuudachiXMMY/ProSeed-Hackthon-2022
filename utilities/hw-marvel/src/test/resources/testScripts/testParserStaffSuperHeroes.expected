# Test Marvel parser to load StaffSuperHeroes.tsv correctly

loaded graph staffHeroes

staffHeroes contains: Ernst-the-Bicycling-Wizard Grossman-the-Youngest-of-them-all Notkin-of-the-Superhuman-Beard Perkins-the-Magical-Singing-Instructor
the children of Ernst-the-Bicycling-Wizard in staffHeroes are: Grossman-the-Youngest-of-them-all(CSE331) Notkin-of-the-Superhuman-Beard(CSE331) Notkin-of-the-Superhuman-Beard(CSE403) Perkins-the-Magical-Singing-Instructor(CSE331)
the children of Grossman-the-Youngest-of-them-all in staffHeroes are: Ernst-the-Bicycling-Wizard(CSE331) Notkin-of-the-Superhuman-Beard(CSE331) Perkins-the-Magical-Singing-Instructor(CSE331)
the children of Notkin-of-the-Superhuman-Beard in staffHeroes are: Ernst-the-Bicycling-Wizard(CSE331) Ernst-the-Bicycling-Wizard(CSE403) Grossman-the-Youngest-of-them-all(CSE331) Perkins-the-Magical-Singing-Instructor(CSE331)
the children of Perkins-the-Magical-Singing-Instructor in staffHeroes are: Ernst-the-Bicycling-Wizard(CSE331) Grossman-the-Youngest-of-them-all(CSE331) Notkin-of-the-Superhuman-Beard(CSE331)



